### PR TITLE
doc/rados/operations: add prompts to operating.rst

### PR DESCRIPTION
**Pull request by zdover23**
This commit adds ".. prompt:: bash $"-style prompts to operating.rst.
This brings this file up to the standard established in 2020 when
Kefu added support for the ".. prompt::" directive.

This commit is a part of an initiative to modernize the presentation
of all BASH commands in the RADOS documentation.

The progress of this project can be tracked here:
https://tracker.ceph.com/issues/57108

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/operations/operating.rst [hunk 1]
```diff
@@ -10,87 +10,107 @@ Running Ceph with systemd
...
+To list the Ceph systemd units on a node, execute:
+
+.. prompt:: bash $
+
+   sudo systemctl status ceph\*.service ceph\*.target
 
 Starting all Daemons
 --------------------
 
 To start all daemons on a Ceph Node (irrespective of type), execute the
```
Comment (anthonyeleven) [inline doc/rados/operations/operating.rst:29]: Suggest `To start all Ceph daemons on a node (irrespective of role), execute the`

File: doc/rados/operations/operating.rst [hunk 2]
```diff
@@ -10,87 +10,107 @@ Running Ceph with systemd
...
-        sudo systemctl stop ceph\*.service ceph\*.target
+.. prompt:: bash $
+
+   sudo systemctl stop ceph\*.service ceph\*.target
 
 
 Starting all Daemons by Type
 ----------------------------
 
 To start all daemons of a particular type on a Ceph Node, execute one of the
```
Comment (anthonyeleven) [inline doc/rados/operations/operating.rst:51]: To start all Ceph daemons of a particular type on a node, execute one of the

File: doc/rados/operations/operating.rst [hunk 3]
```diff
@@ -10,87 +10,107 @@ Running Ceph with systemd
...
-        sudo systemctl start ceph-mds.target
+   sudo systemctl start ceph-osd.target
+   sudo systemctl start ceph-mon.target
+   sudo systemctl start ceph-mds.target
 
 
 Stopping all Daemons by Type
 ----------------------------
 
 To stop all daemons of a particular type on a Ceph Node, execute one of the
```
Comment (anthonyeleven) [inline doc/rados/operations/operating.rst:64]: To stop all Ceph daemons of a particular type on a node, execute one of the

**Review by anthonyeleven — APPROVED** [review]
LGTM overall.  I left a few comments suggesting more specific wording, but that doesn't warrant holding up approval.

**Comment by zdover23** [comment]
> LGTM overall. I left a few comments suggesting more specific wording, but that doesn't warrant holding up approval.

I'll add this to a list of textual changes to be made during a future editorial pass. Expect that editorial pass to begin sometime within the next two weeks, when all of the prompts have been modernized.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/47587 - Pacific backport
https://github.com/ceph/ceph/pull/47586 - Quincy backport

**Pull request merged**